global.json

```diff
@@ -1,15 +1,15 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.7.20366.6",
+    "version": "5.0.100-preview.8.20417.9",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.100-preview.7.20366.6",
+    "dotnet": "5.0.100-preview.8.20417.9",
     "vs": {
       "version": "16.6"
```

Comment:
❔ This needs an update as well right?

Reply:
Yeah, probably. I just want to see the CI work first :)